samples/framework/libraries/migrate-library-csproj-keep-existing/src/Car/Car.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <AssemblyName>Car</AssemblyName>
+    <PackageId>Car</PackageId>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
```

Comment:
Templates are not providing `AssemblyName` or `PackageId`; and in this case, I think the folder name will be used ... so it will be "Car" by default.

`NetStandardImplicitPackageVersion` isn't necessary here. I don't think `PackageTargetFallback` is required either.